deploy/0.0.1/cd-deploy.sh

```diff
@@ -0,0 +1 @@
+tb deploy --yes --populate --fixtures --wait
```

Comment:
If I'm not wrong, when running this in production it'll do a `populate` hence it'll duplicate data in `signal_minute_mv` right?